src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/AtomPub10ServiceDocumentFormatter.cs

```diff
@@ -266,82 +219,62 @@ private static void ReadInlineCategories(XmlReader reader, InlineCategoriesDocum
                         if (reader.IsStartElement(Atom10Constants.CategoryTag, Atom10Constants.Atom10Namespace))
                         {
                             SyndicationCategory category = CreateCategory(inlineCategories);
-                            Atom10FeedFormatter.ReadCategory(reader, category, version, preserveAttributeExtensions, preserveElementExtensions, maxExtensionSize);
+                            Atom10FeedFormatter.ReadCategory(reader, category, version, true, true, maxExtensionSize);
```

Comment:
I find that true/false values like this at call sites are hard to understand, as you don't have any context as to what they mean.  I like in these cases to used named arguments.